guides/micronaut-oauth2-linkedin/kotlin/src/main/kotlin/example/micronaut/LinkedInMe.kt

```diff
@@ -0,0 +1,12 @@
+package example.micronaut
+
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+import io.micronaut.core.annotation.NonNull
+import javax.validation.constraints.NotBlank
+
+@Introspected
+data class LinkedInMe @Creator constructor(
```

Comment:
Is `@Creator` necessary? If you want to keep, we need to add a callout and explain it to the user.